api/graphql/schema.graphql

```diff
@@ -1990,6 +2027,8 @@ type Mutation {
   removePost(postId: ID, slug: String, groupId: ID): GenericResult
   # Remove a Post from a Collection, if you have permission to
   removePostFromCollection(collectionId: ID, postId: ID): GenericResult
+  # Remove a badge/role from a member of a group, inititated by that member or by a group moderator
```

Comment:
oh interesting, the member can remove themselves from a role? I guess that makes sense but worth checking on in product meeting

Reply:
I wanted to leave the option open but we don't need to do the UI for it on the first pass